clients/drcachesim/tools/invariant_checker.cpp

```diff
@@ -364,9 +364,9 @@ invariant_checker_t::parallel_shard_memref(void *shard_data, const memref_t &mem
             report_if_false(shard,
                             !shard->saw_timestamp_but_no_instr_ ||
                                 // The invariant is relaxed for a signal.
-                                // prev_xfer_marker_ is cleared on an instr, so if set to
-                                // non-sentinel it means it is immediately prior, in
-                                // between prev_instr_ and memref.
+                                // prev_xfer_marker_ is cleared on an instr, so if set
+                                // to non-sentinel it means it is immediately prior,
```

Comment:
Does your clang-format have different settings somehow and that is why it ended these lines earlier?